crates/networking/rpc/rpc.rs

```diff
@@ -472,6 +473,16 @@ pub fn map_net_requests(req: &RpcRequest, contex: RpcApiContext) -> Result<Value
     }
 }
 
+pub async fn map_mempool_requests(
+    req: &RpcRequest,
+    contex: RpcApiContext,
+) -> Result<Value, RpcErr> {
+    match req.method.as_str() {
+        "mempool_content" => mempool::content(contex).await,
```

Comment:
This should be `txpool_content` for compatibility (ref https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool#txpool-content)

Reply:
Okay

Reply:
Updated ([use geth namespace for compatibility](https://github.com/lambdaclass/ethrex/pull/2869/commits/39409e0e426ac113c8cc656d7abe9b396c8c819b))